compiled/src/Utility_TEST.cc

```diff
@@ -793,7 +797,7 @@ TEST(MsgsTest, SetSphericalCoordinates)
 TEST(MsgsTest, SetStringMsg)
 {
   msgs::StringMsg msg;
-  msgs::Set(&msg, "a string msg");
+  msgs::Set(&msg, std::string("a string msg"));
```

Comment:
Do we not support passing a `const char *` anymore?

Reply:
I probably just missed it when I was generating the stub conversions, it has been restored.